rust/parquet_derive/Cargo.toml

```diff
@@ -18,8 +18,13 @@
 [package]
 name = "parquet_derive"
 version = "3.0.0-SNAPSHOT"
+license = "Apache-2.0"
+description = "Apache Parquet implementation in Rust"
```

Comment:
I think we should have a more accurate description of what the parquet_derive crate provides rather than use the same description as the main parquet crate.